Source/ParameterEncoding.swift

```diff
@@ -85,13 +85,17 @@ public struct URLEncoding: ParameterEncoding {
         case brackets
         /// No brackets are appended. The key is encoded as is.
         case noBrackets
+        /// Brackets containing the item index appended. This matches the jQuery and Node.js behavior.
+        case indexInBrackets
 
-        func encode(key: String) -> String {
+        func encode(key: String, index: Int) -> String {
```

Comment:
I'd suggest something like `encode(key: String, atIndex index: Int)` to better match Swift's naming guidelines to make the usage a bit more fluent.